liota/core/metric_handler.py

```diff
@@ -30,4 +30,177 @@
 #  THE POSSIBILITY OF SUCH DAMAGE.                                            #
 # ----------------------------------------------------------------------------#
 
-# TODO: Implement this module
+from Queue import Queue, PriorityQueue, Full
+import heapq
+import logging
+from threading import Thread, Condition
+from time import time as _time
+
+from liota.lib.utilities.utility import getUTCmillis
+
```

Comment:
This file lacks necessary updates in my package manager fork. What I had there may not be the best or even a good implementation but better have some sort of implementation or package manager may not work correctly later.

And also, think of a way to close these threads elegantly.